catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -258,7 +258,7 @@ protected Option setupDistribution() {
             "cxf-ws-security",
             "cxf-frontend-javascript",
             "cxf-jaxrs"),
-        SecurityFeatures.start("security-core-api"),
```

Comment:
Installing the bundles individually via `BundleInfo` installs the versions "as in the project", i.e., the versions in the dependency tree for this module. This module depends on `tika-input-transformer`, and that artifact's dependency tree includes version 2.9.8 for a couple of the bundles, so those versions get installed during the test, and the test fails. Installing the bundles via the feature instead ensures the expected versions are installed.